framework/contrib/pyDOE/build_regression_matrix.py

```diff
@@ -16,6 +16,14 @@
 import numpy as np
 
 def grep(haystack, needle):
+    """
+    Grep element from haystack
+    
+    Returns
+    -------
+    start : int
+    
+    """

```

Comment:
Did you update the doc by yourself? If so, could you add more details about the input parameters?

Reply:
yes...pylint was complaining about this docstrings (I am confused on why this has not been captured before from the pyling...did the "testing" settings change?

Reply:
Addressed